coalahtml/_coalahtml/app/controllers/files.js

```diff
@@ -2,37 +2,51 @@
 
 angular.module('coalaHtmlApp')
   .controller('FilesCtrl', function ($scope, $http, $routeParams) {
+    $http.get("data/Constants.json").success(function(content) {
+      $scope.Constants = content;
+      $http.get($scope.Constants.roothome).success(function(content) {
+        $scope.fileName = $routeParams.fileName || content;
+        $scope.roothome = content;
+        parseCoalaProject($scope.fileName, $scope.Constants.data);
+      });
+    });
     var parseCoalaProject = function(fileName, projectDir, filesJSON) {
-      /* Expected file structure in filesJSON:
-       * JSON object with keys as directory relative to the project
-       * path and values are files inside directory relative to the
-       * project.
-       * For files in the project root, the key should be "".
-       */
-      filesJSON = filesJSON || projectDir + '/files.json';
+      /*
+      Expected file structure in filesJSON:
+      JSON object with keys as directory relative to the project
+      path and values are files inside directory relative to the
+      project.
+      For files in the project root, the key should be "".
+      */
```

Comment:
Yeah we agreed to use 2 space indentation for js files. Should I make it 4
space, commit and create a new commit separately for 2 space?
ᐧ

On Mon, May 23, 2016 at 10:34 AM, Attila <notifications@github.com> wrote:

> In coalahtml/_coalahtml/app/controllers/files.js
> <https://github.com/coala-analyzer/coala-html/pull/46#discussion_r64169790>
> :
>
> >      var parseCoalaProject = function(fileName, projectDir, filesJSON) {
> > -      /* Expected file structure in filesJSON:
> > -       * JSON object with keys as directory relative to the project
> > -       * path and values are files inside directory relative to the
> > -       * project.
> > -       * For files in the project root, the key should be "".
> > -       */
> > -      filesJSON = filesJSON || projectDir + '/files.json';
> > +      /*
> > +      Expected file structure in filesJSON:
> > +      JSON object with keys as directory relative to the project
> > +      path and values are files inside directory relative to the
> > +      project.
> > +      For files in the project root, the key should be "".
> > +      */
>
> You change comment layout, also some indents
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/coala-analyzer/coala-html/pull/46/files/1429aeb955ace745709694534f7ebee5991b96df..53a777041ea97ca6462231a5bbf56f655f845db2#r64169790>
>